dcs/action.py

```diff
@@ -1787,6 +1787,25 @@ def dict(self):
         return d
 
 
+class PictureToAll(Action):
+    predicate = "a_out_picture"
+
+    def __init__(self, file_res_key: Optional[ResourceKey] = None):
+        super(PictureToAll, self).__init__(PictureToAll.predicate)
+        if file_res_key:
+            self.file_res_key = file_res_key
```

Comment:
This needs to happen outside the `if`. As written, this object has different members depending on the arguments passed.